index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Share+Tech+Mono&family=Ubuntu+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Molle:ital@1&family=Rajdhani&family=Warnes&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Work</title>
+  </head>
+  <body>
+    <div class="wrap">
+      <div class="header">
+        <h1>
+          <span>My p</span><span>roje</span><span>cts</span><span>!</span>
+        </h1>
+        <h2 class="bottom">If you can imagine it, you can create it</h2>
```

Comment:
This is more like a paragraph, not a heading. Imagine this like a book, it does not make sense to be a h2 :)

Reply:
Yes makes sence!